appyx-navigation/common/src/commonMain/kotlin/com/bumble/appyx/navigation/children/ChildCallback.kt

```diff
@@ -0,0 +1,7 @@
+package com.bumble.appyx.navigation.children
+
+import com.bumble.appyx.navigation.lifecycle.CommonLifecycle
+
+typealias ChildrenCallback<T1, T2> = (commonLifecycle: CommonLifecycle, child1: T1, child2: T2) -> Unit
```

Comment:
I wonder if we need the "Common" prefix in general. (The same question regarding all other expect/actual names).

Btw it has an additional meaning in this context (common = the common minimum, combined from the lifecycles of child1 and child2).

Reply:
I added the prefix to help disambiguate the lifecycle class when bouncing between android and multiplatform code, but that is a bit of an android-centric perspective.

Happy to do whatever regarding prefixes, but could be done in a follow-on ticket on this node-mp branch?